packages/components/select/src/useSelect.ts

```diff
@@ -802,36 +781,30 @@ export const useSelect = (props, states: States, ctx) => {
   }
 
   const toggleMenu = (e?: PointerEvent) => {
-    if (e && !states.mouseEnter) {
+    if ((e && !states.mouseEnter) || selectDisabled.value) {
```

Comment:
**nit:** perhaps we could remove the curly bracket and just append `return` statement after `)`, WDTY?